src/widgets/helper/ChannelView.cpp

```diff
@@ -1316,13 +1316,24 @@ void ChannelView::mouseMoveEvent(QMouseEvent *event)
         {
             auto element = &hoverLayoutElement->getCreator();
             auto thumbnailSize = getSettings()->thumbnailSize;
-            if (thumbnailSize == 0 || isInStreamerMode())
+            if (thumbnailSize == 0)
             {
                 tooltipPreviewImage.setImage(nullptr);
             }
             else
             {
-                tooltipPreviewImage.setImage(element->getThumbnail());
+                if (isInStreamerMode() &&
+                    getSettings()->streamerModeHideLinkThumbnails &&
+                    !element->getThumbnail()->url().string.isEmpty())
```

Comment:
```suggestion
                    getSettings()->streamerModeHideLinkThumbnails)
```
This here will make chatterino not crash :).
Also I don't understand in the first place why you would check if the element has a thumbnail url?

Reply:
> I don't understand in the first place why you would check if the element has a thumbnail url?

I do that to check if link actually has a thumbnail to not display Streamer Mode image for links that have no thumbnail. I was looking on how to do that and found out that links without thumbnail URL has obviously no thumbnail, so I used that to determine whether we should display the static image or not. Maybe there's a better way to do that, but this is just a quick solution I found yesterday.

Reply:
Should be fixed by e0f24d7.